package.json

```
@@ -88,6 +88,6 @@
 	},
 	"engines": {
 		"node": "^12.22.0 || ^14.17.0 || >=16.0.0",
-		"npm": ">=6"
+		"npm": ">=8"
```

This could throw errors in projects that use `eslint-plugin-testing-library` (especially when they use Yarn)

```suggestion
		"npm": ">=6"
```

---

Good point!